crates/xxi-node/src/message_handler.rs

```diff
@@ -432,18 +443,23 @@ impl TryFrom<Message> for TenTenOneMessage {
             Message::Channel(ChannelMessage::Reject(reject)) => {
                 TenTenOneMessage::Reject(TenTenOneReject { reject })
             }
-            Message::OnChain(_) | Message::SubChannel(_) => bail!("Unexpected dlc message"),
+            Message::OnChain(_)
+            | Message::SubChannel(_)
+            | Message::Channel(ChannelMessage::RenewOffer(_))
+            | Message::Channel(ChannelMessage::SettleOffer(_))
+            | Message::Channel(ChannelMessage::Offer(_)) => {
+                unreachable!()
+            }
```

Comment:
👍 I think I don't mind the `unreachable!`, but we should change the function signature to be infallible then.